sdks/python/apache_beam/options/pipeline_options_test.py

```diff
@@ -104,6 +104,13 @@ def tearDown(self):
        'display_data': [
            DisplayDataItemMatcher('mock_multi_option', ['op1', 'op2'])]
       },
+      {'flags': ['--flink_master=testmaster:8081', '--parallelism=42'],
```

Comment:
It feels odd to put flink-specific tests here.

Reply:
As far as I see this also tests other Runner Options here, e.g. Dataflow. So it seemed fine to me.